src/backend/tests/test-data/projects.test-data.ts

```diff
@@ -244,6 +244,14 @@ export const manufacturer2: Manufacturer = {
   materials: [material1]
 };
 
+export const manufacturer3: Manufacturer = {
+  name: 'Manufacturer3',
```

Comment:
can you change the name to be an actual manufacturer (of your choosing)

Reply:
The two test data right above the one I added also have Manufacturer1 and Manufacturer2 for the names, should I change those as well

Reply:
done